README.md

```diff
@@ -27,6 +27,7 @@ Before working with the Jekyll code, you will need to install Jekyll:
     $ gem install jekyll
     $ gem install jekyll-redirect-from
     $ gem install html-proofer
+    $ gem install activesupport -v 4.2.7
```

Comment:
BTW, while investigating syntax highlighting dependencies, I've just discovered how the Gemfile works: http://jekyll.tips/jekyll-casts/gemfiles-and-the-bundler/

Ours is currently forcing a specific version of activesupport for reasons I don't understand. Will dig in more tonight and let you know what I find.